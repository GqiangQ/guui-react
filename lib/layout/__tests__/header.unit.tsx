import {configure} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Header from '../header';

configure({adapter: new Adapter()});

describe(Header.displayName, () => {
  it('DisplayName', () => {
    expect(Header.displayName).toEqual('Header');
  });
  describe('props: className', () => {
    it('= xxx', () => {
      const tree = renderer
        .create(<Header className="xxx"></Header>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('props: style', () => {
    it(`= {color: 'red'}`, () => {
      const tree = renderer
        .create(<Header style={{color: 'red'}}></Header>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
