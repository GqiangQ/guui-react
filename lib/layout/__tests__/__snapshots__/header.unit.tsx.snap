// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header props: className = xxx 1`] = `
<div
  className="gu-header xxx"
/>
`;

exports[`Header props: style = {color: 'red'} 1`] = `
<div
  className="gu-header "
  style={
    Object {
      "color": "red",
    }
  }
/>
`;
